quattor/types/system.pan

```diff
@@ -40,6 +147,22 @@ include 'quattor/types/aii';
 include 'quattor/types/grid';
 include 'quattor/types/hardware';
 
+type structure_system_aquilon = {
+    "advertise_status" ? boolean
+    "archetype"     ? structure_archetype
+    "build"         ? string with match (SELF, 'build|blind|ready|failed|install')
+    "security"      ? structure_security
+    "users"         ? nlist
+    "eon_ids"       ? long[]
+    "eon_id_maps"   ? nlist
+    "owner_eon_id"  ? long
+    "provides"      ? nlist
+    "services"      ? structure_services{}
+    "personality"   ? structure_personality
+    "host_environment" ? string
+    "features"      ? nlist
+}
```

Comment:
missing `;`, causing failure in https://jenkins1.ugent.be/job/NCM_Components-pull_requests/705/org.quattor.cfg.module$opennebula/console